controllers/authenticationController.js

```diff
@@ -0,0 +1,21 @@
+const bcrypt = require('bcrypt');
+const userModel = require('../models/userModel');
+const registerUser = async(req, res) => {
+  const{firstName,lastName,username,password,confirmPassword,email,isAdmin} = req.body;
```

Comment:
isAdmin will not come from payload, in case of this API, it will be true @ashok-choudhary-0